dataland-backend/src/main/kotlin/org/dataland/datalandbackend/utils/JsonExampleFormattingConstants.kt

```diff
@@ -29,4 +29,19 @@ object JsonExampleFormattingConstants {
         "\"InvestmentFirm\": " +
         taxonomyEligibilityActivity +
         "}"
+
+    private const val procurementCategory = "{" +
+        "\"definitionProductTypeService\": [\"string\"]," +
+        "\"numberOfSuppliersPerCountryCode\": {" +
+        "\"DE\": null," +
+        "\"GB\": 2" +
+        "}," +
+        "\"orderVolume\": 0" +
```

Comment:
Reminder to change the field names here too for the next changes. (Just in case the "Refactoring" function of the IDE does not already detect it).

Reply:
Did that

Reply:
👍